bravado/mapping/response.py

```diff
@@ -0,0 +1,30 @@
+class ResponseLike(object):
+    """
+    Define a common interface for bravado to interface with client side
+    response objects from various 3rd party libraries.
+
+    Subclasses are responsible for providing attrs for __required_attrs__.
+    """
```

Comment:
So my understanding of this class is that its the `Response` interface that bravado expects, and can also be used as a  subclass to provide more friendly error messages to developers trying to implement this interface.

I usually prefer to avoid inheritance and let them debug the attribute error, but I'm ok with this too.